examples/influxdb_data_model.ipynb

```diff
@@ -557,15 +556,17 @@
     "\n",
     "for job_id in jobs['ids']:\n",
     "    \n",
-    "    data = requests.get(SQUASH_API_URL + \"/job/{}\".format(job_id)).json()\n",
+    "    if job_id > 5244:\n",
```

Comment:
What's this magic number for?

Reply:
oops, should not have committed that line.